specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/Jobs.json

```diff
@@ -1126,7 +1126,7 @@
           "description": "List of container definitions for the Container Apps Job.",
```

Comment:
Could you consider having a new job execution file with all its changes?

Reply:
There's no change on the API level. It's actually the same container definition but unified.